app/models/customer.py

```diff
@@ -0,0 +1,30 @@
+from flask import current_app
+from app import db
+# from datetime import datetime
+from sqlalchemy.orm import backref, relationship
+from sqlalchemy.sql.schema import ForeignKey
+from sqlalchemy import Table, Column, Integer, ForeignKey 
+
+# from flask_migrate import Migrate
+
+
+class Customer(db.Model):
+    __tablename__ = "customers"
+
+    customer_id = db.Column(db.Integer, primary_key=True,autoincrement=True)
+    name = db.Column(db.String)    
+    register_at = db.Column(db.DateTime, nullable=True) # nullable=True ??? 
+    postal_code = db.Column(db.String) 
+    phone = db.Column(db.String) 
+    videos_checked_out_count =  db.Column(db.Integer, default=0)
```

Comment:
Instead of this being a column you can use the rentals to count that.  

To do that it's helpful to add a relationship:
```suggestion
    rentals = db.relationship('Rental', back_populates='customer', lazy=True)
```